CODEOWNERS

```diff
@@ -10,8 +10,11 @@ azure-pipelines.yml  @openvinotoolkit/openvino-admins
 # QA Tests:
 /tests/  @openvinotoolkit/openvino-tests-maintainers
 
-# IE in general:
+# IE Core:
 /inference-engine/  @openvinotoolkit/openvino-ie-maintainers
+/inference-engine/src/transformations/  @GlebKazantaev @ichuraev
```

Comment:
just ask to add into the @openvinotoolkit/openvino-ie-maintainers

Reply:
Ok, I will try to join to this group.

Reply:
I don't want someone from openvino-ie-maintainers approve changes in this path except me and Ilya.

Reply:
Why don't you create openvino-transformation-maintainers with 2 people?